food-truck-api/src/main/java/food/truck/api/notification/NotificationService.java

```diff
@@ -17,7 +17,6 @@
 import java.time.Instant;
 import java.util.List;
 import java.util.Optional;
-import java.util.concurrent.atomic.AtomicBoolean;
```

Comment:
Thanks for removing this.

Reply:
I don't recall removing it, so I'll assume either Kevin or my delinter did it.